src/sentry/integrations/slack/actions/notification.py

```diff
@@ -122,41 +128,56 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
                 rule_action_uuid=rule_action_uuid,
             )
 
-            # We need to search by rule action uuid and rule id, so only search if they exist
-            reply_broadcast = False
-            thread_ts = None
-            if (
-                OrganizationOption.objects.get_value(
-                    organization=self.project.organization,
-                    key="sentry:issue_alerts_thread_flag",
-                    default=ISSUE_ALERTS_THREAD_DEFAULT,
-                )
-                and rule_action_uuid
-                and rule_id
-            ):
-                parent_notification_message = None
+            def get_thread_ts(lifecycle: EventLifecycle) -> str | None:
```

Comment:
Are there any existing tests covering this?

Reply:
Yep, `tests.sentry.integrations.slack.actions.notification.test_slack_notify_service_action.TestInit.test_after_*`.